test/Microsoft.ML.IntegrationTests/Training.cs

```diff
@@ -22,7 +23,7 @@ public Training(ITestOutputHelper output) : base(output)
         /// <summary>
         /// Training: It is easy to compare trainer evaluations on the same dataset.
         /// </summary>
-        [Fact]
+        [NativeDependencyFact("CpuMathNative")]
```

Comment:
What about this test relies on CpuMathNative? This doesn't feel right - the code should work the same regardless of CpuMathNative or not.

Reply:
Adding on to my comment below about not all parts of CPU Math having a fallback. This test uses the factorization machine. Here is the relevant code:
https://github.com/dotnet/machinelearning/blob/1b3cb77b9752fe4279376039ee20fc42822e4845/src/Microsoft.ML.CpuMath/FactorizationMachine/FactorizationMachineInterface.netcoreapp.cs#L33-L36

arm doesn't support AVX, and the only other option is to use the native code in CPU math, which uses the x86/x64 SIMD instructions.

I have added an issue, #5871, to track this. IIRC this is the only case in CPU Math that doesn't have a basic managed fallback.

Reply:
Let's not do this with `[NativeDependencyFact("CpuMathNative")]` then. Let's mark these with a new `FieldAwareFactorizationMachineFact` attribute instead. That way it more clearly describes the intent here - the FieldAwareFactorizationMachine feature doesn't work outside of x64/x86. Then we can link to your new issue in that attribute saying why it doesn't support arm.

Reply:
Makes sense. I'll go ahead and make that change as well.